openapi/alfresco/Module.md

```diff
@@ -7,8 +7,8 @@ This Alfresco Content Services REST APIs enable you to access to the core featur
 
 Before using this connector in your Ballerina application, complete the following:
 
-* Create a [Alfresco Account](https://www.alfresco.com/try-alfresco-acs).
-* Obtain tokens by following [this guide](https://docs.alfresco.com/content-services/latest/develop/repo-ext-points/authentication/).
+* Create an [Alfresco Account](https://www.alfresco.com/try-alfresco-acs).
+* Once registered, you will receive an email with instructions to set up your Alfresco environment, including the default username and password. Use these Basic Authentication credentials to access the Alfresco Content Services API.
 
 ## Quickstart
```

Comment:
Shall we follow the same structure as the revamped connectors for quick-start guide ?

https://github.com/ballerina-platform/module-ballerinax-stripe?tab=readme-ov-file#quickstart